data/json/items/bionics.json

```diff
@@ -290,7 +290,7 @@
     "type": "BIONIC_ITEM",
     "name": { "str": "Directional EMP CBM" },
     "looks_like": "bio_int_enhancer",
-    "description": "Small parabolic EMP field generators that are mounted in the palm of the user's right hand.  These can be used to instantly fire a wide, but short-ranged blast which will disable electronics and robots.",
+    "description": "A small parabolic EMP field generator that is designed to be mounted in the user's left arm and lead down to the palm of the hand.  It can be used to instantly fire a wide, but short-ranged blast to heavily damage electronics and robots.",
```

Comment:
```suggestion
    "description": "A small parabolic EMP field generator that is designed to be mounted in the user's left arm and leads down to the palm of the hand.  It can be used to instantly fire a wide, but short-ranged blast to heavily damage electronics and robots.",
```